frontend/packages/kubevirt-plugin/src/models/templates/vm-template-yaml.ts

```diff
@@ -10,20 +10,21 @@ metadata:
   name: vm-template-example
   labels:
     kubevirt.io/os: fedora27
-    miq.github.io/kubevirt-is-vm-template: 'true'
     template.kubevirt.io/type: vm
```

Comment:
template details will not show base template without `vm.kubevirt.io/template` label